lib/devicetrust/native/device_windows.go

```diff
@@ -78,106 +79,69 @@ func handleTPMActivateCredential(encryptedCredential, encryptedCredentialSecret
 	return windowsDevice.handleTPMActivateCredential(encryptedCredential, encryptedCredentialSecret)
 }
 
-// getDeviceSerial returns the serial number of the device using PowerShell to
-// grab the correct WMI objects. Getting it without calling into PS is possible,
-// but requires interfacing with the ancient Win32 COM APIs.
+// getDeviceSerial returns the serial number of the device.
 func getDeviceSerial() (string, error) {
-	cmd := exec.Command(
-		"powershell",
-		"-NoProfile",
-		"Get-WmiObject Win32_BIOS | Select -ExpandProperty SerialNumber",
-	)
-	// ThinkPad P P14s:
```

Comment:
I would keep the powershell comments, it helps double-checking the stuff. Same for others.

Reply:
Ack. I had the same thought.